src/app/map/[id]/State/types.ts

```diff
@@ -1,26 +1,47 @@
 import type { MapItemAPIContract } from "~/server/api/types/contracts";
 import {
-  HexCoordSystem,
+  CoordSystem,
   type HexCoord,
+  HexDirection,
 } from "~/lib/domains/mapping/utils/hex-coordinates";
 // import type { ScaleState } from "./scale"; // Commented out problematic import
 
+// TODO: Move this to a shared constants file
```

Comment:
You are right, it should live in its own constant file. This file does not exist yet, create it at src/app/map/[id]/constants.ts. Update the src/app/map/[id]/README.md file with this information.